.github/workflows/purge-build-artifacts.yml

```diff
@@ -0,0 +1,16 @@
+name: Purge build artifacts
+
+on:
+  schedule:
+    # Every sunday at 1am
+    - cron: '0 1 * * 0'
+
+jobs:
+  remove-old-artifacts:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Remove old artifacts
+        uses: c-hive/gha-remove-artifacts@v1
```

Comment:
This one is new to me. Does Github hold on to artifacts automatically? Or would this be paired with a setting?

Reply:
Github will auto hold on to artifacts but with a customizable retention period - removing.

https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#artifact-and-log-retention-policy

> For private, internal, and GitHub Enterprise repositories: you can change this retention period to anywhere between 1 day or 400 days.